src/mlpack/methods/linear_regression/linear_regression.cpp

```diff
@@ -85,6 +85,7 @@ void LinearRegression::Train(const arma::mat& predictors,
       lambda * arma::eye<arma::mat>(p.n_rows, p.n_rows);
 
   parameters = arma::solve(cov, p * r.t());
+  return ComputeError(predictors,responses);
```

Comment:
Tiny style issue---there should be a space between `predictors,` and `responses`.

Reply:
Okay fixed it!